pymongo/pool.py

```diff
@@ -407,6 +411,32 @@ def _cond_wait(condition: threading.Condition, deadline: Optional[float]) -> boo
     return condition.wait(timeout)
 
 
+def _get_timeout_details(options: PoolOptions) -> dict[str, float]:
+    details = {}
+    timeout = _csot.get_timeout()
+    socket_timeout = options.socket_timeout
+    connect_timeout = options.connect_timeout
+    if timeout:
+        details["operationTimeoutMS"] = timeout * 1000
```

Comment:
Suggest remaining this to "timeoutMS" to match the URI option name. operationTimeoutMS doesn't appear anywhere else so this could be confusing.

Reply:
Agreed it should match the URI option name. Is `timeoutMS` generally specific enough for users, or should we consider changing that option name to reflect its functionality better?